src/elements/container/sticky-bar/sticky-bar.visual.spec.ts

```diff
@@ -0,0 +1,94 @@
+import { SbbBreakpointMediumMin } from '@sbb-esta/lyne-design-tokens';
+import { setViewport } from '@web/test-runner-commands';
+import { html, type TemplateResult } from 'lit';
+
+import {
+  describeEach,
+  describeViewports,
+  visualDiffDefault,
+  visualRegressionFixture,
+} from '../../core/testing/private.js';
+import { waitForLitRender } from '../../core/testing.js';
+
+import './sticky-bar.js';
+import '../container.js';
+import '../../action-group.js';
+import '../../button.js';
+import '../../link.js';
+import '../../title.js';
+
+describe(`sbb-sticky-bar`, () => {
+  let root: HTMLElement;
+
+  const cases = {
+    color: [undefined, 'white', 'milk'],
+    containerExpanded: [false, true],
+    scrolled: [false, true],
+  };
+
+  const containerContent = (): TemplateResult => html`
+    <sbb-title level="4">Example title</sbb-title>
+    <p class="sbb-text-s">The container component will give its content the correct spacing.</p>
+    <sbb-secondary-button style="margin-block-end: 0.75rem;" size="m"
+      >See more</sbb-secondary-button
+    >
+  `;
+
+  const actionGroup = (): TemplateResult => html`
+    <sbb-action-group align-group="stretch" orientation="vertical" style="width:100%;">
+      <sbb-block-link align-self="start" icon-name="chevron-small-left-small">
+        Link
+      </sbb-block-link>
+      <sbb-button>Confirm</sbb-button>
+    </sbb-action-group>
+  `;
+
+  describeViewports(() => {
+    describeEach(cases, ({ color, containerExpanded, scrolled }) => {
+      beforeEach(async function () {
+        const element = await visualRegressionFixture(
+          html`
+            <div id="scroll-container" style="overflow: auto; height: 400px;">
+              <sbb-container
+                ?expanded=${containerExpanded}
+                style="--sbb-container-background-color: var(--sbb-color-cloud)"
+              >
+                ${containerContent()} ${containerContent()} ${containerContent()}
+                <p>Content end</p>
+                <sbb-sticky-bar .color=${color}> ${actionGroup()} </sbb-sticky-bar>
+              </sbb-container>
+            </div>
+          `,
+          { padding: '0' },
+        );
+
+        root = element.querySelector('#scroll-container')!;
+
+        if (scrolled) {
+          root.scrollTop = root.scrollHeight;
+          await waitForLitRender(root);
+        }
+      });
+
+      it(
+        visualDiffDefault.name,
+        visualDiffDefault.with((setup) => {
+          setup.withSnapshotElement(root);
+        }),
+      );
+    });
+  });
+
+  it(
+    `viewport=medium_short content`,
+    visualDiffDefault.with(async (setup) => {
+      await setViewport({ width: SbbBreakpointMediumMin, height: 400 });
```

Comment:
sadly this could causing webkit to fail in some cases. I suggest to use the same logic as it internally does.
```suggestion
      this.ctx['requestViewport'] = { width: SbbBreakpointMediumMin, height: 400 }
```

Reply:
Just for info, what problem do we have with Webkit?

ps. I don't think the solution would work (it is already too late to set the ctx variable)

Reply:
Ok, then the solution would maybe be to move the viewport after the fixture creation. 

The problem is, that Safari does not apply all media queried variables correctly if the viewport is already set when the fixture is being created. But, it's reproducible: if you see, that your screenshots look fully correct, it's ok like you did.

Reply:
It seems fine but to be sure I'll move the setViewport after the fixture :)